sdk/src/manifest_store.rs

```diff
@@ -169,6 +169,18 @@ impl ManifestStore {
                 .map(|_| Self::from_store(&store, &mut validation_log))
         })
     }
+
+    pub async fn from_manifest_and_asset_bytes_async(
```

Comment:
This is a public API function, yes? If so, can we please add proper documentation for it?

Reply:
Yes - of course. Just wanted to get this out for preliminary review.